mm/dynamic/c/Magazine.c

```
@@ -1,12 +1,53 @@
+// Runtime knobs
+static volatile int g_validation_level = KHEAP_VALIDATION_NONE; // 0=none,1=basic,2=full
+static volatile int g_stats_enabled = 1; // can be toggled by perf mode
+
+static inline void StatsAdd(uint32_t cpu, int sc, volatile uint64_t* ctr) {
+    // Best-effort increment; tearing is acceptable for stats.
+    if (!g_stats_enabled) return;
+    (void)cpu; (void)sc;
+    (*((uint64_t*)ctr))++;
+}
+
+static void StatsSlabAllocated(int sc) {
```

Clang (14.0.6)

[warning] parameter name 'sc' is too short, expected at least 3 characters

(readability-identifier-length)